PR: https://github.com/odoo/odoo/pull/

From: f90290d1c290c91056306900bdafc6dc418bee65
From: Denis Vermylen (dve)
Date: 2016-08-31 15:17:37

Structural Changes: 7
Total Changes: 13

[IMP] website,w_sale: Website Admin settings view

- Add optional products and pricelists to ecommerce section
- add mail template sent when confirming a sale order
- rename and clarify using radio widgets

================================= pseudo patch: =================================

--- a/addons/website/models/res_config.py
+++ b/addons/website/models/res_config.py
@@ -30,8 +30,15 @@ class WebsiteConfigSettings(models.TransientModel):
     cdn_activated = fields.Boolean('Use a Content Delivery Network (CDN)', related='website_id.cdn_activated')
     cdn_url = fields.Char(related='website_id.cdn_url')
     cdn_filters = fields.Text(related='website_id.cdn_filters')
-    module_website_form_editor = fields.Boolean("Form builde = create and customize forms")
-    module_website_version = fields.Boolean("A/B testing and versioning")
+    module_website_form_editor = fields.Boolean("Form builder")
+    module_website_form_editor = fields.Selection([
+        (0, 'Use standard forms'),
+        (1, 'Create and customize forms to generate emails, leads, issues and extra information in the checkout process (new snippet available)')
+        ], "Form Builder")
+    module_website_version = fields.Selection([
+        (0, 'No version management and A/B testing (easy)'),
+        (1, 'Allow multiple versions of the same page (advanced)')
+        ], "A/B Testing")
     favicon = fields.Binary('Favicon', related='website_id.favicon')
     
     # Set as global config parameter since methods using it are not website-aware. To be changed

--- a/addons/website/models/website.py
+++ b/addons/website/models/website.py
@@ -164,7 +164,7 @@ class Website(models.Model):
     domain = fields.Char('Website Domain')
     company_id = fields.Many2one('res.company', string="Company", default=lambda self: self.env.ref('base.main_company').id)
     language_ids = fields.Many2many('res.lang', 'website_lang_rel', 'website_id', 'lang_id', 'Languages', default=_active_languages)
-    default_lang_id = fields.Many2one('res.lang', string="Default language", default=_default_language)
+    default_lang_id = fields.Many2one('res.lang', string="Default Language", default=_default_language)
     default_lang_code = fields.Char(related='default_lang_id.code', string="Default language code", store=True)
 
     social_twitter = fields.Char('Twitter Account')
